src/C4Console.cpp

```diff
@@ -598,7 +598,7 @@ bool C4Console::In(const char *szText)
 		// done
 		return true;
 	}
-	// begins with '#'? then it's a message. Route cia ProcessInput to allow #/sound
+	// begins with '#'? then it's a message. Route via ProcessInput to allow #/sound
```

Comment:
This has nothing to do with this PR. It could be in an extra spelling-fixes-commit.